syntax = "proto3";

package cws.common;

message ReturnMsg {
	int32 err_code = 1;
	string err_msg = 2;
}

message RpcContext {
	string service_name = 1;
	string method_name = 2;
	int32 content_length = 3;
}

enum Gender {
	MALE = 0;
	FEMALE = 1;
}
